app/players/dwarves/dwarf_warrior.py

```diff
@@ -0,0 +1,17 @@
+from app.players.dwarves.dwarf import Dwarf
+
+
+class DwarfWarrior(Dwarf):
+    def __init__(self,
+                 nickname: str,
+                 favourite_dish: str,
+                 hummer_level: int) -> None:
+        super().__init__(nickname, favourite_dish)
+        self._hummer_level = hummer_level
+
+    def player_info(self) -> str:
+        return (f"Dwarf warrior {self.nickname}. {self.nickname} "
+                f"has a hummer of the {self._hummer_level} level")

```

Comment:
In the `player_info` method, the phrase 'has a hummer of the {self._hummer_level} level' could be improved for clarity. Consider rephrasing it to 'has a hammer with a level of {self._hummer_level}' for better readability.